src/styles/blocks/cards.scss

```diff
@@ -0,0 +1,25 @@
+.cards {
+  padding: 50px 40px;
+
+  &__container {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+  }
+
+  &__row {
+    display: flex;
+  }
+
+  &__row--gap {
+    margin-bottom: 46px;
+  }
+
+  &__col--gap {
+    margin-right: 48px;
+  }
```

Comment:
Don't use modifiers like this. Add margins to all cards and then remove where needed using child pseudo-classes.
Page should work correctly with any amount of cards.